roles/katello_pulp3/tasks/main.yml

```diff
@@ -0,0 +1,97 @@
+---
+- name: create static dir
+  file:
+    path: '/var/lib/pulp/static/'
+    state: directory
+    mode: '777'
+
+- include_role:
+    name: "{{ item }}"
+  with_items:
+    - pulp-database
+    - pulp-workers
+    - pulp-resource-manager
+    - pulp-content
+  vars:
+    pulp_content_host: "{{ ansible_fqdn }}"
+    pulp_default_admin_password: password
+    pulp_requirements_dir: "https://raw.githubusercontent.com/pulp/pulpcore/master/"
+    pulp_plugin_source_dir: "git+https://github.com/pulp/pulpcore-plugin.git#egg=pulpcore-plugin"
+    pulp_source_dir: "git+https://github.com/pulp/pulpcore.git#egg=pulpcore"
+    pulp_install_plugins:
+      pulp-file:
+        app_label: "file"
+        source_dir: "git+https://github.com/pulp/pulp_file.git#egg=pulp-file"
+      pulp-docker:
+        app_label: "docker"
+        source_dir: "git+https://github.com/pulp/pulp_docker.git#egg=pulp-docker"
+      pulp-ansible:
+        app_label: "ansible"
+        source_dir: "git+https://github.com/pulp/pulp_ansible.git#egg=pulp-ansible"
+
+    pulp_secret_key: "unsafe_default"
+    pulp_db_type: postgres
+    pulp_user: pulp
+    pulp_install_dir: /usr/local/lib/pulp
+
+- name: correct published permissions
+  file:
+    path: '/var/lib/pulp/published/'
+    state: directory
+    owner: pulp
+    group: apache
+
+- name: create artifact dir
+  file:
+    path: '/var/lib/pulp/artifact'
+    state: directory
+    owner: pulp
+    group: pulp
+
+- name: Deploy reverse proxy configuration
+  template:
+    src: templates/pulp-reverse-proxy.conf.j2
+    dest: /etc/httpd/conf.d/05-foreman-ssl.d/pulp-reverse-proxy.conf
+    owner: apache
```

Comment:
Unless apache needs to write to these files I'd say the owner/group should be root. That goes for all these conf.d files.